diff.py

```diff
@@ -255,6 +255,20 @@ def complete_symbol(
         help="""Automatically update when source/object files change.
         Recommended in combination with -m.""",
     )
+    parser.add_argument(
+        "-0",
+        "--zeroway",
+        dest="zeroway",
```

Comment:
this can use the same dest as -b/-3 with another `const`, since it's in the same category as those (and incompatible with them)